### ISSUE TITLE
Problems encountered in training resnet_v2 from scratch with your code on slim

### ISSUE THREAD

**Issue by tianye2017resnet** [tensorflow/models#5384]
System information
•What is the top-level directory of the model you are using: slim
•Have I written custom code: no
•OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 16.04
•TensorFlow installed from (source or binary): binary
•TensorFlow version (use command below): 1.9.0.
•Bazel version (if compiling from source): -
•CUDA/cuDNN version: CUDA 9
•GPU model and memory: Tesla V100, 16GB
•Exact command to reproduce: training and evaluating /home/linkface/sunyi/models/research/slim/nets/resnet_v2.py.

Describe the problem

I have recently run your code to train the resnet_v2_50 and resnet_v2_101 model from scratch which was downloaded from slim.I have tried to set the batch size to 32,64,128 and 256 and set the image size to 224 and 299.All other parameters remain unchanged from the code.However,both the top1 accuracy and the top5 accuracy are 2 or 3 points lower than your results.
I am wondering if you could kindly send me your settings to achieve the results and the necessary information about it.
Thanks.

**Comment by gmt710** [comment]
yes,I met this situation too.

**Comment by gmt710** [comment]
I met this situation too.

**Comment by shiqi1994** [comment]
I met this situation too. Did you find the solution?

**Comment by kumariko** [comment]
@tianye2017resnet We are checking to see if you still need help on this issue? We recommend that you upgrade to 2.7 which is latest stable version of TF . Also please have a look on the [link1](https://stackoverflow.com/questions/39774449/fully-convolutional-resnets-using-tf-slim-run-very-slow) and let us know if it helps? Thanks!
